platform-dom/src/main/java/com/softicar/platform/dom/input/auto/DomAutoCompleteList.java

```diff
@@ -12,7 +12,7 @@
  */
 public class DomAutoCompleteList implements Iterable<String> {
 
-	public static final int MAXIMUM_ELEMENT_COUNT = 16;
+	public static final int MAXIMUM_ELEMENT_COUNT = 17;
```

Comment:
generally, this seems to be ok, although the fact the here we have the constant 17 and in JS, we have 16 will be confusing for anybody how looks at it in the future